Diminish Bloom's (#38)
data-lessons/instructor-training: _episodes/15-lesson-study.md

- "A good exercise provides useful guidance to instructors about next steps needed in teaching."
---

Yesterday we learned many cognitive principles and teaching practices that are at the heart of the curriculum we design, collaboratively, in The Carpentries
community. Ultimately, however, this curriculum is in *your* hands. If you have signed up to teach Carpentries workshops, we know that you care about the
content you are teaching, and we suspect you probably have a pretty good idea of how to prepare your technical knowledge. We also expect that you may be more

> of a novice when it comes to implementing the principles and practices we discussed yesterday in a real classroom.

Reviewer: More of a novice than what? I think you mean more of a novice than you are about the technical content. Can we make that clearer?

Author: Thanks! I think this is better now?

When you prepare to teach a workshop, it is useful to take a moment *before* deeply reviewing your technical content to think through the learner experience, how
your teaching will translate into their learning, and how both of you will know when that happens.

### Anticipating Your Audience

To teach effectively, you have to know *who* you are teaching. You may have a broad idea about the type of audience you expect. You may
(we hope!) have a plan to learn a few things about your participants after a class has begun. However, in thinking about your learners, it is 
also important to consider the broader contexts they bring in ways that you will never get to fully explore in your classroom. It can be helpful
to think through ways in which learners' experiences and needs may be similar to or different from your own, or from each others. 

For our next exercise, we will explore some deep thinking about the 'whole people' who might come to your classroom by creatively brainstorming a **learner 
profile**. This is a good way to support an empathic and intentional approach to your plan for instruction.

> 

Reviewer: I like this framing!

Reviewer 2: me too!

- "A good exercise provides useful guidance to instructors about next steps needed in teaching."
---

Yesterday we learned many cognitive principles and teaching practices that are at the heart of the curriculum we design, collaboratively, in The Carpentries
community. Ultimately, however, this curriculum is in *your* hands. If you have signed up to teach Carpentries workshops, we know that you care about the
content you are teaching, and we suspect you probably have a pretty good idea of how to prepare your technical knowledge. We also expect that, compared with your 
technical accomplishments, your skill level may be less advanced when it comes to implementing the principles and practices we discussed yesterday in a real classroom. 

> Even if you have years of experience in the classroom, new instructional practices can take you into uncharted territory! 

Reviewer 2: this tone is a little weird? what's the goal here

Author: Thanks -- completely changed this section. Purpose is to emphasize teaching as a skill and justify investment of time and effort in preparing instructional goals as well as technical content.

When you prepare to teach a workshop, it is useful to take a moment *before* deeply reviewing your technical content to think through the learner experience, how
your teaching will translate into their learning, and how both of you will know when that happens.

### Anticipating Your Audience

To teach effectively, you have to know *who* you are teaching. You may have a broad idea about the type of audience you expect. You may
(we hope!) have a plan to learn a few things about your participants after a class has begun. However, in thinking about your learners, it is 
also important to consider the broader contexts they bring in ways that you will never get to fully explore in your classroom. It can be helpful
to think through ways in which learners' experiences and needs may be similar to or different from your own, or from each others. 

For our next exercise, we will explore some deep thinking about the 'whole people' who might come to your classroom by creatively brainstorming a **learner 
profile**. This is a good way to support an empathic and intentional approach to your plan for instruction.

Learner profiles have three parts:
- the person's general background,

> - the problem they face,

Reviewer 2: maybe the motivating problem they face? to make it more specific?

Author: ```suggestion
- the motivating problem they face,
```

> ## Learner Profiles
>

> > Sketch out a profile of someone you might expect to attend your first workshop, based on the example above.

Reviewer: Consider removing "first". Sometimes instructors have taught workshops before taking instructor training.

today? In the next exercise, we are going to take a moment to understand those objectives, because in the best cases they are quite specific about exactly what 
level a learner is expected to operate at. This can be helpful in both designing formative assessments and in evaluating potential additions or digressions for 
appropriateness.

When thinking about learning objectives, it is important to keep the 'level' of a learner in mind. There are many ways of conceptualizing 'level' in education. 
However, you probably have your own sense of what kinds of goals can be achieved by a novice, and what kinds of things can
only be achieved after some 'groundwork' has been laid. 

> ## Evaluate Learning Objectives
>
> Select one learning objective from the episode you've used for teaching practice. Copy it into the etherpad
> then add numbers below your objective to address the following:
>
> 1. Identify the *action* specified in the learning objective. 
> 2. Suppose a learner had met this objective. Write a "next step" learning objective for them on the same topic. How is the action different?

> > 3. Is it possible to write an objective at a *lower* level than the one in your lesson? What would that look like?

Reviewer: Will "next step" and "lower level" be clear enough without going into Bloom's first?

Author: This is really the crux of this restructuring, and I don't know for sure. My sense about Blooms is that 90% of the problem people have with it is that it has an intimidating name. The heirarchy itself is, I believe, sufficiently intuitive that it can reasonably be derived on its own. That being said, these kinds of constructivist activities frequently do go wrong. 
The question isn't really whether they will get it 'right' per Blooms, but whether they will be able to come up with anything at all to compare. 
A more guided alternative to this structuring could be to present pieces of the Taxonomy without naming it at first.

Author: I tweaked the language here to at least make it clear that we want them to use their own intuition. Also adds a learner situation to each prompt (learner wants to do more/is struggling) to help them imagine a response.

>
> This exercise should take about 10 minutes.  
{: .challenge}

Chances are, your class has re-invented many components of Bloom's Taxonomy. While Bloom's is not an empirically derived framework, research supports the 
existence -- and constraints -- of such a heirarchy. 

![A six level pyramid labelled from bottom to top: remember, understand, apply, analyze, evaluate, create.](../fig/Bloom_taxonomy.png)

What is useful about this model is that it helps us to be mindful of skipping steps in our ambition to achieve the highest levels of learning. Many courses
strive -- and brag about -- bringing learners to "higher order" thought processes, asking learners to apply, analyze, evaluate, or create using content they

> learn in a course. When these efforts fail, it is typically because instructors have neglected to attend to the 'lowly' task of helping learners to remember and

Reviewer: Not sure that the text will be able to convey the right tone for "lowly".

understand the underlying concepts. Most Carpentries workshop objectives, since they address novices, will focus at these lower levels. Where more advanced 

> actions are expected, be mindful to provide support where prior learning may be required.

Reviewer: Consider removing "Where more advanced actions are expected" . I think this is covered by the "where prior learning may be required."

> This exercise should take about 10 minutes.  
{: .challenge}

More exmples of learner profiles for Software Carpentry are provided [here]({{ site.swc_site }}/audience/). Note that some of these example profiles contain more
personal details, which can be uncomfortable to read. It is not necessary to include these kinds of details when creating your own learner profiles. However this kind of

> thinking can also support a careful approach to accessibility and inclusion. Whether through learner profiles or more general brainstorming, it is

Reviewer 2: i like this, but maybe we should tag the people who commented on the issue about this exercise in pretty soon?

advanced in Wiggins and McTighe's *[Understanding by Design](http://www.worldcat.org/title/understanding-by-
design/oclc/56491025)*. The order of course design in this case becomes

1.  Determine learning objectives
2.  Decide what constitutes evidence that objectives have been met, and design assessments to target that evidence
3.  Design instruction: Sort assessments in order of increasing complexity, and write content that connects everything together

This process is much more likely to lead to an appropriately stepped curriculum that guides learners to specific goals and demonstrates to both instructors
and learners that those goals have been met. 

In the case of Carpentries workshops, the lesson has already been designed, and pre-written exercises are often available to use for assessment. here, **reverse 
instructional design** principles might be applied as follows: 

1. Review the lesson's learning objectives carefully, thinking about how they will work for your audience
2. Scan the lesson to identify promising points to check in with your learners, using formative assessment to verify that objectives have been met, and then 

> 3. Review the connecting content in detail to be sure everything works and you have anticipated likely problems and questions.

Reviewer 2: 🎉  this is really helpful, i think

## Working With Learning Objectives

> All Carpentries workshops should have learning objectives listed at the top of each episode. Did you notice the objectives in the lesson you chose to study for 

Reviewer 2: ```suggestion
All Carpentries lessons should have learning objectives listed at the top of each episode. Did you notice the 
objectives in the lesson you chose to study for 
```

> today? In the next exercise, we are going to take a moment to understand those objectives, because in the best cases they are quite specific about exactly what 
> level a learner is expected to operate at. This can be helpful in both designing formative assessments and in evaluating potential additions or digressions for 
> appropriateness.

Reviewer 2: ```suggestion
today? In the next exercise, we are going to take a moment to understand those objectives, because in the best 
cases they are quite specific about exactly what a learner is expected to be able to do by the end of the workshop.
This can be helpful in both designing additional formative assessments and in evaluating potential additions or digressions for 
appropriateness.
```

removing "level" because bloom's levels haven't been introduced prior.

{: .challenge}

> ## "Do You Understand?" is a Bad Question
>
> Learners are often reluctant to admit when they do not understand. Furthermore, self-assessments of skill in the absence of formative assessment are often inaccurate
> because of the [Dunning-Kruger effect](https://en.wikipedia.org/wiki/Dunning%E2%80%93Kruger_effect):
> the less people know about a subject,
> the less accurate their estimate of their knowledge is. Therefore, if you ask a room full of people
> "Do you understand?" the result will invariably be a number of 'yes' responses (many of them inaccurate) which tend to drown out a
> variable amount of silence. Instead, a targeted formative assessment takes the inaccuracy and stress of self-judgement away and demonstrates to all whether understanding has

> > met the instructor's goal.

Reviewer: "understanding has met the instructor's goal" Understanding as the subject in this clause makes it a bit confusing I think.
